modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/numeric/NumericDDMFormFieldTemplateContextContributorTest.java

```diff
@@ -267,17 +274,33 @@ public void testGetSymbolsBrazilLocale() {
 		Assert.assertEquals(".", symbols.get("thousandsSeparator"));
 	}
 
-	private DDMFormFieldRenderingContext _createDDMFormFieldRenderingContext() {
+	@Test
+	public void testGetValue() {
+		_ddmFormField.setProperty("localizedObjectField", true);
+
 		DDMFormFieldRenderingContext ddmFormFieldRenderingContext =
-			new DDMFormFieldRenderingContext();
+			createDDMFormFieldRenderingContext();
 
-		ddmFormFieldRenderingContext.setLocale(_locale);
+		String value = JSONUtil.put(
+			"en_US", "5"
+		).put(
+			"pt_BR", "9.47"
+		).toString();
 
-		return ddmFormFieldRenderingContext;
+		ddmFormFieldRenderingContext.setValue(value);
+
+		Map<String, Object> parameters =
+			_numericDDMFormFieldTemplateContextContributor.getParameters(
+				_ddmFormField, ddmFormFieldRenderingContext);
+
+		Assert.assertEquals(value, String.valueOf(parameters.get("value")));
```

Comment:
Can we use `MapUtil.getString()` and inline `parameters` ?

Reply:
Done!